song_rater/models/song.py

```diff
@@ -1,6 +1,17 @@
 from django.db import models
+from django.core.validators import MaxValueValidator
 
 
 class Song(models.Model):
     title = models.CharField(max_length=100)
     artist = models.CharField(max_length=100)
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(fields=['title', 'artist'], name='unique_song')
+        ]
+
+
+class Rating(models.Model):
+    song = models.ForeignKey(Song, on_delete=models.CASCADE, related_name='ratings')
+    rating = models.PositiveSmallIntegerField(validators=[MaxValueValidator(5)])
```

Comment:
Is `0` or `1` supposed to be your minimum rating value?

Reply:
Your commit message explains that `0` is the intended minimum, which is exactly what this does.

Reply:
I added a minimum value validator to make 1 the minimum value as in the story